src/index.ts

```diff
@@ -62,8 +69,9 @@ export function provideAnswersHeadless(config: HeadlessConfig): AnswersHeadless
   const stateManager = new ReduxStateManager(
     store, headlessId || DEFAULT_HEADLESS_ID, headlessReducerManager);
   const httpManager = new HttpManager();
+  const httpHeaders = getAdditionalHttpHeaders(additionalHttpHeaders);
```

Comment:
would it be bad if we did this inside the AnswersHeadless constructor instead of in provideAnswersHeadless? It makes things slightly less flexible, but also lets us cut down on the number of interfaces we're exporting for this feature? We won't need to export both HeadlessAdditionalHttpHeaders, and ALSO AdditionalHttpHeaders. I think that might result in an overall nicer developer experience. It'll also match Rose's spec more closely

Reply:
do you mean have `provideAnswersHeadless` pass `additionalHttpHeaders` directly to the `AnswersHeadless` constructor and have the constructor call the `getAdditionalHttpHeaders` method?
I'm not sure how that would cut down on the imports? we would still need to export both `HeadlessAdditionalHttpHeaders` (as part of `HeadlessConfig`) and `AdditionalHttpHeaders` because it's extended by `HeadlessAdditionalHttpHeaders` and referenced in Core docs

Reply:
OH I see I thought we wouldn't need to export AdditionalHttpHeaders if we were only extending it, but api-extractor means we have to

Reply:
could you check with Rose on the interface naming then? I think there's a chance she'll have some thoughts on shortening some of them. I was thinking if we could export less then we could simplify the names some, but it looks like that's not possible

Reply:
specifically I feel like `ClientSDKHeaderValuesExcludingHeadless` is a little wordy? at the very least maybe something like `HeadlessClientSdkHeaderValues`?

Reply:
sure, I'll ask her